README.md

```diff
@@ -16,7 +16,37 @@ This project is built using the [ASP.NET MVC](http://www.asp.net/mvc) web framew
    cd ivr-phone-tree-csharp
    ```
 
-2. Build the solution
+1. Build the solution
```

Comment:
Could you add a period at the end of the line? Please take a look at http://www.mhhe.com/mayfieldpub/tsw/period.htm.